numpy/distutils/ccompiler.py

```diff
@@ -701,14 +701,18 @@ def CCompiler_cxx_compiler(self):
                             "Intel C Compiler for 32-bit applications on Windows")
 compiler_class['intelemw'] = ('intelccompiler', 'IntelEM64TCCompilerW',
                               "Intel C Compiler for 64-bit applications on Windows")
+compiler_class['clang-cl'] = ('clang_cl', 'ClangCL',
+                              "Clang CL Compiler for 64-bit Windows")
```

Comment:
Doesn't clang-cl support 32-bit compilation?
```suggestion
                              "Clang CL Compiler for Windows")
```